assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -173,6 +177,11 @@ export function LeftRail() {
                 <NavLink
                   active={path.value === state.currentPath}
                   href={resolveRoute(path.value)}
+                  {...(path.isExternal && {
+                    rel: 'noreferrer',
+                    target: '_blank',
+                    onClick: (evt) => onExternalLinkClick(evt, path),
+                  })}
```

Comment:
I believe this could fail if this results to `path.isExternal && {...}` actually becomes `false`.

We should do `path.isExternal ? { ... } : {}`